api/server/app.h

```diff
@@ -125,6 +125,7 @@ extern Model model;
 extern Log log;
 extern Graph graph;
 extern pg_error *pg_error;
+
```

Comment:
I'm not sure about the purpose of that new line 😄 About everything else, nice job it's really more explicit now 👍